src/services/liking.service.js

```diff
@@ -13,6 +13,19 @@ const LikingService = {
     }
     like.update({ isLiked: !like.isLiked });
     return sendResult(res, 200, ` The accommodation like  status updated to ${like.isLiked}`, like);
+  },
+
+  async getLikes(req, res) {
```

Comment:
This is an operation with the database. We do not need to refer to `req`or `res`inside this method. Just get the count of all the `likes` belonging to a particular accommodation